apps/snitch_core/lib/core/tools/ecto_enums/ecto_enums.ex

```diff
@@ -0,0 +1,2 @@
+import EctoEnum
+defenum(StateEnum, draft: 0, active: 1, in_active: 2, deleted: 3)
```

Comment:
Can the Enum module be renamed something like `ProductStateEnum`?

Reply:
done